src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -263,10 +330,34 @@ class DialogZWaveJSAddNode extends LitElement {
                       class="link"
                       @click=${this._chooseInclusionStrategy}
                     >
-                      Advanced inclusion
+                      ${this.hass.localize(
+                        "ui.panel.config.zwave_js.add_node.choose_inclusion_strategy"
+                      )}
                     </button>
                   </p>
                 </div>
+                ${this._supportsSmartStart
+                  ? html` <div class="outline">
+                      <h2>
+                        ${this.hass.localize(
+                          "ui.panel.config.zwave_js.add_node.qr_code"
+                        )}
+                      </h2>
+                      <ha-svg-icon .path=${mdiQrcodeScan}></ha-svg-icon>
+                      <p>
+                        ${this.hass.localize(
+                          "ui.panel.config.zwave_js.add_node.qr_code_paragraph"
+                        )}
+                      </p>
+                      <p>
+                        <mwc-button @click=${this._scanQRCode}>
```

Comment:
Just a tip from my experience creating big and large complex dialog flows: inline the event handlers. It keeps logic and UI close, easier to keep track of what's going on.

Reply:
Ideally we would just break the dialog up, inlinling the event handlers means we create them on every render, also not ideal

Reply:
Yeah I thought about that, break it up and then have mixins to bring them all together again. Re-creating them on every render is not end of the world as this dialog is not a performance bottleneck.